cycle_submit.py

```diff
@@ -273,16 +279,20 @@ def main():
 	else:
 		password = options.password
 
+        if options.hostname.startswith('http'):
```

Comment:
We should not be overloading https:// into the hostname argument. If we called it --host, it should not have a protocol. But it should also not have a port, so maybe this fits in with the previous poor choice.